ush/forecast_postdet.sh

```diff
@@ -663,14 +667,17 @@ WW3_nml() {
 }
 
 WW3_out() {
+  mkdir -p "${COM_CONF}"
   echo "SUB ${FUNCNAME[0]}: Copying output data for WW3"
 }
 
 
 CPL_out() {
+  mkdir -p "${COM_CONF}"
   echo "SUB ${FUNCNAME[0]}: Copying output data for general cpl fields"
   if [[ "${esmf_profile:-}" = ".true." ]]; then
-    ${NCP} "${DATA}/ESMF_Profile.summary" "${COM_ATMOS_HISTORY}/ESMF_Profile.summary"
+    ${NCP} "${DATA}/ESMF_Profile.summary" "${COM_ATMOS_HISTORY}/ESMF_Profile.summary" # TODO: I don't think this is needed for cycling.
```

Comment:
```suggestion
```

Reply:
Done.

Reply:
I think the TODO statement can be removed here

Reply:
@JessicaMeixner-NOAA I think my push crossed your query. Can you update/refresh and see if these issues still persist?